### PR TITLE
feat(dev): Do not write CPATH and LDFLAGS in .zshrc

### PR DESCRIPTION
**Pull request by armenzg**
First of all, updating it to correct values in the future is complicated.

Second, it had hard coded versions of the libraries. This means that future brew installations
could make it useless.

Exporting it as variables within the `make install-py-dev` target seems sufficient.

The negative of this approach means that `pip install` of these packages without using the make target would fail.

### CODE REVIEW

**Comment by armenzg** [comment]
@jan-auer let me know what your thoughts are

File: scripts/lib.sh [hunk 1]
```diff
@@ -113,6 +113,11 @@ install-py-dev() {
         # This saves having to install postgresql on the Developer's machine + using flags
         # https://github.com/psycopg/psycopg2/issues/1286
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
+        BREW_PATH="$(brew --prefix)/opt"
+        # The CPATH is needed for confluent-kakfa --> https://github.com/confluentinc/confluent-kafka-python/issues/1190
+        export CPATH="${BREW_PATH}/librdkafka/include"
```
Comment (jan-auer) [inline scripts/lib.sh:118]: _nit_: If you inline `$(brew --prefix librdkafka)` here, it also checks whether the package is still installed and you could even catch it down to print a nice error message.

**Review by jan-auer — APPROVED** [review]
Looks good to me 👍

File: scripts/lib.sh (hunk 1 above)
Comment (armenzg) [reply scripts/lib.sh:118]: I like it! Thanks!

**Pull request merged**